app/forms/support_interface/application_forms/edit_applicant_details_form.rb

```diff
@@ -43,6 +49,9 @@ def save!
         @application_form.phone_number = phone_number
         @application_form.audit_comment = audit_comment
```

Comment:
Minor nit but rather than doing: 

```
candidate.x = y
candidate.a = b
candidate.save!

```
we can just do 

`candidate.update!(x: y, a: b)`

Reply:
All changes made, will merge when able 👍